### PR TITLE
[WIP] feat: add flex

### PR DESCRIPTION
**Pull request by LYXOfficial**

### CODE REVIEW

**Review by SnowballXueQiu — APPROVED** [review]

**Comment by GaoNeng-wWw** [comment]
Please run `pnpm changeset` generate changeset file

File: packages/components/flex/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "name": "@qwqui/center",
+    "version": "1.0.1-alpha.0",
```
Comment (GaoNeng-wWw) [inline packages/components/flex/package.json:3]:
```suggestion
    "version": "0.0.0",
```

**Review by GaoNeng-wWw — CHANGES_REQUESTED** [review]

File: packages/components/flex/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+    "name": "@qwqui/center",
```
Comment (GaoNeng-wWw) [inline packages/components/flex/package.json:2]:
```suggestion
    "name": "@qwqui/flex",
```

File: packages/components/flex/src/flex.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+                alignItems: props.align,
+                flexDirection: props.direction,
+                flexWrap: props.wrap,
+            }}
+            className={classes.root + " " + (props.className ?? "")}
+        >
+            {props.children}
+        </div>
+    );
+}
```
Comment (GaoNeng-wWw) [inline packages/components/flex/src/flex.tsx:38]:
```suggestion
}
Flex.displayName='@qwqui/layout/flex'
```

File: packages/components/index.ts
Comment (GaoNeng-wWw) [inline packages/components/index.ts:1]: Please do not push `index.ts` file

**Comment by GaoNeng-wWw** [comment]
Please resolve change request before run `pnpm changeset` generate changeset file

**Comment by LYXOfficial** [comment]
review大佬

**Comment by SnowballXueQiu** [comment]
还有什么问题吗目前

**Comment by GaoNeng-wWw** [comment]
请运行`pnpm changeset`来生成changeset文件

**Review by GaoNeng-wWw — APPROVED** [review]

**Pull request merged**